deployment/provision_vagrant_vm.sh

```diff
@@ -1,8 +1,19 @@
 #! /usr/bin/env bash
 
+docker=false
+
 set -x # print executed commands
 
-USER="evap"
+if [ "$1" = "docker" ]
+then
+    docker=true
+    echo "provisioning docker"
+    USER="vagrant"
```

Comment:
? In docker, the user is called vagrant? How did that happen?

Reply:
for other usernames provisioning fails